pkg/kubectl/describe.go

```diff
@@ -1012,6 +1012,15 @@ func describeContainerEnvVars(container api.Container, resolverFn EnvVarResolver
 		none = "\t<none>"
 	}
 	w.Write(LEVEL_2, "Environment Variables:%s\n", none)
+
+	for _, e := range container.EnvFrom {
+		if len(e.Prefix) == 0 {
+			w.Write(LEVEL_3, "%s\tConfigMap\n", e.ConfigMapRef.Name)
```

Comment:
I think these need to be further qualified or broken out so that it's clear that all keys in the configmap are going to get injected.